### PR TITLE
Update MIPs README

### PR DESCRIPTION
**Pull request by CPSTL**
The goal of this PR is to bring attention to the much-needed MIPs repo README update and allow us to contribute to it out in the open. Hopefully, it will also make the update come sooner :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,160 +1,81 @@
...
-|   9  |   2020-05-02 | [MIP9](https://github.com/makerdao/mips/tree/master/MIP9)                          |Community Greenlight (Collateral Onboarding) |
-|   10 |   2020-05-02 | [MIP10](https://github.com/makerdao/mips/tree/master/MIP10)                         |Oracle Management |
-|   11 |   2020-05-02 | [MIP11](https://github.com/makerdao/mips/tree/master/MIP11)                         |General Risk Model Management (Collateral Onboarding) |
-|   12 |   2020-05-02 | [MIP12](https://github.com/makerdao/mips/tree/master/MIP12)                         |Collateral and Risk Parameter Management |
+MIPs have two types: **technical** if they propose changes or additions to smart contracts code related to the Maker Protocol; or **general** otherwise.
 
+### Subproposals
 
-## Join the conversation 
+Subproposals are meta-processes of general, recurring processes defined in MIPs. Subproposals are created and used as a means of creating a process to build a process.
```
Comment (Davidutro) [inline README.md]: Most people will not understand what you mean when using the phrase "meta-processes" & may also be confused by the "used as a means of creating a process to build a process" bit.

Here's my proposal to make it more simplified, let me know if you feel like this is not technically accurate in some way;

"Subroposals are a common recurring process found in numerous MIPs. They are created and used as a means of proposing and enacting decisions nested within specific MIPs."

What do you think @CPSTL @blimpa

File: README.md [hunk 2]
```diff
@@ -1,160 +1,81 @@
...
+- Post the draft [on the appropriate forum section](https://forum.makerdao.com/c/mips/14)
+- Submit it to the [GitHub MIP repository](https://github.com/makerdao/mips)
+
+After the two above tasks have been completed, MIP Editors will help assign a number for the proposal. The proposed MIP (or subproposal - they are treated the same way in terms of how to propose them) will then enter a period of community feedback (most frequently referred to as Request For Comments or RFC). This is a great opportunity to be proactive and interact with the broader community, take suggestions, and improve your proposal. Once the feedback period is over and you've incorporated your final changes, there will be a one-week period before you can formally submit the proposal for voting. Lastly, your proposal will go through a voting period. 
+
+If all goes well, by the end of the process you will have contributed a piece of MakerDAO legislation. And that's the gist of it! 
+
+For a more detailed breakdown of the procedure, please read [MIP0c3](http://159.203.86.45:8000/mips/details/MIP0#mip0c3-the-mip-lifecycle).
```
Comment (Davidutro) [inline README.md]: Is the 

>http://159.203.86.45:8000/mips/details/MIP0#mip0c3-the-mip-lifecycle

Is the link here intentional? Should we use 

>https://mips.makerdao.com/mips/details/MIP0#mip0c3-the-mip-lifecycle

instead?

**Review by Davidutro — COMMENTED** [review]
This looks MUCH BETTER than the old README. Thank you for bringing us into the modern-day ser 🙇 

Couple of minor comments/questions below, otherwise looks solid and we should merge asap 🔥

File: README.md (hunk 1 above)
Comment (blimpa) [reply README.md]: First two words that come to my mind are "instance [of a process]" and "actualization", both of which are probably not much of an improvement over "meta-processes"? Maybe something like "documents upon whose approval processes are executed" or some other paraphrase?

Comment (Davidutro) [reply README.md]: "Subroposals are a common recurring process. They are created and used as a means of proposing and enacting decisions nested within specific MIPs."

**Comment by Davidutro** [comment]
fixed link and made that sentence a little more readable. Merging now, since there was not much other feedback

**Pull request merged**